tests/data/test_entryexitanalysis.py

```diff
@@ -140,6 +140,7 @@ def test_backtest_analysis_nomock(default_conf, mocker, caplog, testdatadir, use
     )
     start_analysis_entries_exits(args)
     captured = capsys.readouterr()
+    assert "Analysing on exit signals." not in captured.out
```

Comment:
checking for "non-existance" of a log or output message seems kinda difficult.
While i understand what you're trying to assert here - i'm not sure it's reliable.

i've found numerous instances in the past where the log-message was slightly changed - and the `not` test passed, simply because of a typo fix, not because it's actually testing what it should (if i change the print statement, this test will not bother).

If we assume this to be useful, you may want to extract this (and the corresponding line below) into a constant.
Alternatively, remove this assert - as i'd not see it as THAT useful (or necessary).